includes/migration-guide/retargeting/winforms/accessibility-improvements-windows-forms-controls.md

```diff
@@ -3,7 +3,7 @@
quot; color defined by the OS.</li><li><code>T:System.Windows.Forms.ToolStripMenuItem</code> controls with a <code>P:System.Windows.Forms.ToolStripItem.Checked</code> property set to <em>true</em> now render the associated check mark in a contrasting system color.  Previously the check mark color was not contrasting enough and not visible in High Contrast themes.</li></ul>NOTE: Windows10 has changed values for some high contrast system colors. Windows Forms Framework is based on the Win32 framework. For the best experience, run on the latest version of Windows and opt in to the latest OS changes by adding an app.manifest file in a test application and uncommenting the following code:<pre><code>&lt;!-- Windows 10 --&gt;&#13;&#10;&lt;supportedOS Id=&quot;{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}&quot; /&gt;&#13;&#10;</code></pre><strong>Improved keyboard navigation</strong><ul><li>When a <code>T:System.Windows.Forms.ComboBox</code> control has <code>P:System.Windows.Forms.ComboBox.DropDownStyle</code> set to <code>F:System.Windows.Forms.DropDownStyle.DropDownList</code> and is the first control in the tab order on the form, it now displays a focus rectangle when the parent form is opened using the keyboard. Before this change, keyboard focus was on this control, but a focus indicator was not rendered.</li></ul><strong>Improved Narrator support</strong><ul><li>The <code>T:System.Windows.Forms.MonthCalendar</code> control has added support for assistive technologies to access the control, including the ability for Narrator to read the value of the control when previously it could not.</li><li>The <code>T:System.Windows.Forms.CheckedListBox</code> control now notifies Narrator when the <code>P:System.Windows.Forms.CheckState</code> property has been changed. Previously, Narrator did not recieve notification and as a result users would not be informed that the <code>P:System.Windows.Forms.CheckState</code> had been updated.</li><li>The <code>T:System.Windows.Forms.LinkLabel</code> control has changed the way it notifies Narrator of the text of in the control. Previously, Narrator announced this text twice and read &quot;&amp;&quot; symbols as real text even though they are not visible to a user. The duplicated text was removed from the Narrator announcements, as well as unnecessary &quot;&amp;&quot; symbols.</li><li>The <code>T:System.Windows.Forms.DataGridViewCell</code> control types now correctly report the read-only status to Narrator and other assistive technologies.</li><li>Narrator is now able to read the System Menu of child windows in [Multiple-Document Interface]~/docs/framework/winforms/advanced/multiple-document-interface-mdi-applications.md) applications.</li><li>Narrator is now able to read <code>T:System.Windows.Forms.ToolStripMenuItem</code> controls with a <code>P:System.Windows.Forms.ToolStripItem.Enabled</code> property set to <em>false</em>. Previously, Narrator was unable to focus on disabled menu items to read hte content.</li></ul>|
```

Comment:
Broken link: one of the menubar-control-type links doesn't have an .md file extension